### PR TITLE
Docs: Document `crossOrigin` option

### PR DESCRIPTION
**Pull request by delbaoliveira**

### CODE REVIEW

File: docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: crossOrigin
+description: Use the `crossOrigin` option to add a crossOrigin tag on the `script` tag for pages. Learn more about it here.
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx:3]: Ay.

```suggestion
description: Use the `crossOrigin` option to add a crossOrigin tag on the `script` tags generated by next/script. 
```

File: docs/03-pages/02-api-reference/03-next-config-js/crossOrigin.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: crossOrigin
+description: Use the `crossOrigin` option to add a crossOrigin tag on the `script` tag for pages. Learn more about it here.
```
Comment (delbaoliveira) [inline docs/03-pages/02-api-reference/03-next-config-js/crossOrigin.mdx:3]:
```suggestion
description: Use the `crossOrigin` option to add a crossOrigin tag on the `script` tags generated by `next/script` and `next/head`.
```

File: docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+Use the `crossOrigin` option to add a [`crossOrigin` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin) in all `<script>` tags generated by the <AppOnly>[`next/script`](/docs/app/building-your-application/optimizing/scripts) component</AppOnly> <PagesOnly>[`next/script`](/docs/pages/building-your-application/optimizing/scripts) component and [`next/head`](/docs/pages/api-reference/components/head)components</PagesOnly>, and define how cross-origin requests should be handled.
+
+```js filename="next.config.js"
+module.exports = {
+  crossOrigin: 'anonymous',
+}
```
Comment (styfle) [inline docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx:13]: I didn't realize we had this. I wonder if we should also have this config work with `next/image` too?

File: docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: crossOrigin
+description: Use the `crossOrigin` option to add a crossOrigin tag on the `script` tags generated by `next/script`. 
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+Use the `crossOrigin` option to add a [`crossOrigin` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin) in all `<script>` tags generated by the <AppOnly>[`next/script`](/docs/app/building-your-application/optimizing/scripts) component</AppOnly> <PagesOnly>[`next/script`](/docs/pages/building-your-application/optimizing/scripts) component and [`next/head`](/docs/pages/api-reference/components/head)components</PagesOnly>, and define how cross-origin requests should be handled.
```
Comment (timneutkens) [inline docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx:8]: The option adds `crossorigin` to all Next.js internal script tags, i.e. the runtime and other bundles being loaded, it doesn't add it to `next/image` and `next/head` automatically.

File: docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+```js filename="next.config.js"
+module.exports = {
+  crossOrigin: 'anonymous',
+}
+```
+
+## Options
+
+- `false`: No `crossOrigin` attribute is added to the `script` tags.
```
Comment (timneutkens) [inline docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx:18]:
```suggestion
```

This option doesn't do anything and is going away in a PR I have open.

Comment (delbaoliveira) [reply docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx:18]: Ok, will remove too 👍🏼

File: docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx (hunk 4 above)
Comment (delbaoliveira) [reply docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx:8]: Sorry, the comment in the [source code](https://github.com/vercel/next.js/blob/b5155e166b44aa1a6730826d2f136a8a338c36ec/packages/next/src/server/config-shared.ts#L674) also says:

> Add `"crossorigin"` attribute to generated `<script>` elements generated by `<Head />` or `<NextScript />` components

Should we update the comment?

**Review by leerob — APPROVED** [review]

**Comment by leerob** [comment]
Looks like a Prettier issue on the crossOrigin file 🙏

**Review by leerob — APPROVED** [review]

**Pull request merged**